wweval/R/supplement_analyses_and_figs.R

```diff
@@ -0,0 +1,728 @@
+get_plot_scores_and_forecasts <- function(scores_single_loc_date,
+                                          eval_output_subdir,
+                                          n_calib_days = 10) {
+  this_location <- scores_single_loc_date |>
```

Comment:
The implicit pull confused me for a bit. `unique(scores_single_loc_date$location)` is probably better.

Reply:
Either way works for me. I defer to @kaitejohnson here. @damonbayer's base R approach does avoid the need for

```R
dplyr::distinct(.data$location)
```

which is what this will need to be to avoid R CMD check complaints...

Reply:
Absolutely. But _added_ code should still aim to be compliant.